applications/asset_tracker_v2/doc/sensor_module.rst

```diff
@@ -104,13 +104,11 @@ The sensor module supports integration with the BSEC signal processing library u
 If enabled, the BSEC library is used instead of the BME680 Zephyr driver to provide sensor readings from the BME680 for temperature, humidity, and atmospheric pressure.
 In addition, the BSEC driver provides an additional sensor reading, indoor air quality (IAQ), which is a metric given in between 0-500 range, which estimates the air quality of the environment.
 
-As the BSEC library requires a separate license, it is not a default part of |NCS|, but can be downloaded externally and imported into the |NCS| source tree.
+Please note that using the BSEC library requires accepting a separate license agreement: https://github.com/boschsensortec/Bosch-BSEC2-Library.
```

Comment:
```suggestion
.. note::
   Using the BSEC library requires accepting a separate license agreement: https://github.com/boschsensortec/Bosch-BSEC2-Library.
```
Add BSEC license agreement to the links.txt and use it as guided here https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#external-links

``.. _`BSEC license agreement`: https://github.com/boschsensortec/Bosch-BSEC2-Library`` (this is how it should be in links.txt, under the heading github)
``  `BSEC license agreement`_`` (this is how to use the link in the doc)

Reply:
done